components/zcash_address/src/lib.rs

```diff
@@ -256,7 +254,7 @@ impl ZcashAddress {
     /// [`encode`]: Self::encode
     /// [`Display` implementation]: core::fmt::Display
     /// [`address.to_string()`]: alloc::string::ToString
-    pub fn convert_if_network<T: TryFromRawAddress>(
+    pub fn convert_if_network<T: TryFromAddress>(
```

Comment:
Document this change to required trait bounds in the changelog. (I'm going to edit my prior suggestion to include this.)